app/views/posts/index.html.erb

```diff
@@ -6,7 +6,7 @@
   <% @posts.each do |post| %>
     <article>
       <p>
-        <%= post.content %>
+        <%= link_to("#{ post.content}", post_path(post)) %>
```

Comment:
- [ ] **[OPTIONAL]** To DRY our views, we can create a [partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for posts, then using [collection rendering](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections) we can use it as so:
```
<%= render @posts %>
```
This will search for a partial file named `post` to render its contents.